jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
+  labels:
+    - add: {{ .overrides.label }}
+{{end}}
 fields:
-  summary: {{ or .overrides.summary .fields.summary }}
+  summary: "{{ or .overrides.summary .fields.summary }}"
```

Comment:
Curious on the rational here?  The quotes seem unnecessary unless you want to add meta characters to the summary?  Perhaps a summary of "foo: bar" will throw off the yaml parser?